src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -985,6 +1000,7 @@ private void OnLibraryLoaded(LibraryLoadedEventArgs e)
 
         public static class Categories
         {
+            public const string BuiltIn = "BuiltIn";
             public const string BuiltIns = "Builtin Functions";
```

Comment:
It is used at DynamoCore->Dynamo.Engine.FunctionDescriptor.Category.get()->GetRootCategory() which I found seems to be inconsistent at some point after re-categorization. Because functionDescriptors no longer have "Builtin Functions" as categories. I think we can safely replace this one with the new string, but I will keep it as a follow-up PR in case that breaks any thing.